src/main/java/org/openhab/automation/jrule/internal/handler/JRuleHandler.java

```diff
@@ -284,6 +284,8 @@ private void createRuleInstances() {
         } catch (ClassNotFoundException | IllegalAccessException | NoSuchMethodException | InstantiationException
                 | InvocationTargetException e) {
             logger.error("Could not instantiate JRuleItems file", e);
+        } catch (Error e) {
+            logger.error("Could not instantiate JRuleItems file", e);
```

Comment:
This two catches can be merged together because they do the same handling. Same for the next two changes.